lib/task.js

```diff
@@ -309,17 +309,24 @@ function factory(
         var self = this;
 
         self.instantiateJob();
+        // Support schedulerOverrides for backwards graph definition compatibility
+        var timeout = self.definition.options.timeout;
```

Comment:
I think a descriptive name might be better than `__` as it seems a little odd to me to have a python-style internal variable in a JSON definition. Maybe that's just me though.